bin/run-prod.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -ex
 
 ./bin/run-common.sh
-exec gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
+exec newrelic-admin run-program gunicorn standup.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
```

Comment:
To test I did

* `make rebuild`
* `docker-compose up prod`

This runs the production config and everything worked well.